x-pack/metricbeat/scripts/msetlists/main.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/elastic/beats/metricbeat/scripts/msetlists/gather"
```

Comment:
Nit. For packages with commands I'd prefer to have the package in `github.com/elastic/beats/metricbeat/scripts/msetlists`, and the command in `github.com/elastic/beats/metricbeat/scripts/msetlists/cmd`.
But this case is a bit special, so as you prefer.